### DISCUSSION TITLE
How to write "multi - find & replace " script ?

### DISCUSSION THREAD

**Discussion by connoryang331** [SilentVoid13/Templater#931] [Help]
There's no multi- find & replace plugin, 

So, I find Templater plugin, How to write "multi - find & replace " script ?  , such as :

find " English " ,  replace to " Chinese". 
find " Help " ,  replace to " SOS". 
find " World " ,  replace to " Heaven". 
find " discussion" ,  replace to " Talk show". 

Thanks.

**Comment by AB1908** [comment]
You may be better off using VS Code or any other editor with find and replace capabilities

**Reply by connoryang331** [reply]
I don't know any other editor can do this : once find to replace all ..

**Reply by FelipeRearden** [reply]
There is a plug-in called Apply Patterns that might be helpful for deal with your issue.

https://github.com/jglev/obsidian-apply-patterns-plugin

I hope it helps 🙏

**Reply by connoryang331** [reply]
Thank you @FelipeRearden  so much ! That's what I want .
